### ISSUE TITLE
Contribute to the gallery

### ISSUE THREAD

**Issue by fallenleaves404** [MarcSkovMadsen/awesome-streamlit#28]
Hi, I also want to contribute my interactive project to the gallery!

Web: https://nyc-airbnb-analysis.herokuapp.com/
Github: https://github.com/fallenleaves404/NYC_Airbnb_Analysis

If I need some revision, please let me know!

And also thanks to @lyqht, for inspiring me to build a project

**Comment by lyqht** [comment]
Oh wow, I'm surprised that my mediocre project could inspire you to build one 😄 Streamlit's capabilities are really amazing!

Some feedback from me about your project:
- title of graphs should be more consistent (some are above the plot, some below the x-axis)
- units of your plots (e.g. price in __$)
- num of house in diff area plot extends out of the page boundary
- the interact example is beautiful! ❤️ 
- the navigation items could be better named 

nice!

**Comment by fallenleaves404** [comment]
Yeah, your project really help me a lot! I'm new to GitHub and don't know how to organize the repository. I learn many things from your project!

Thanks for your feedback! I have improved the points you mention

**Comment by MarcSkovMadsen** [comment]
I am on holiday the next week. But @lyqht and @fallenleaves404 its so inspiring to see your conversion, so maybe you can help each other move this forward.

👍
